BAU: fix broken atp

martin-bucinskas opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes

### What changed

- Fixing linting errors that were commited before
- Updating the identity evidence to use sessionData

### Why did it change

The identity evidence needed an update to use sessionData.
There were some linting errors commited.

## Checklists

### Environment variables or secrets

<!-- Delete if changes DO include new environment variables or secrets -->
- [x] No environment variables or secrets were added or changed

martin-bucinskas committed:
BAU: Fixing broken ATPs.

- Some refactoring and fixing the linting which wasn't run before.
- Removing functions within functions and replacing them with arrow functions.
- Using sessionData in req.session.

martin-bucinskas committed:
BAU: Automatic linting updates.

- Ofcourse the linter decides to update the files after they were committed.

AmritSidhu reviewed (dismissed):
Way much nicer and cleaner to read 👍🏽

martin-bucinskas committed:
BAU: Removing the named functions and replacing them for arrow functions.

- Refactor, removing the named functions and replacing them with lambda arrow functions.

AmritSidhu commented on app/features/atp/information/ui/idx/controller.ts:
```diff
@@ -104,24 +104,36 @@ const getInfo = (req: Request, res: Response): void => {
+  const addressCounty = informationAttributes
+    ? informationAttributes.addressCounty
+    : null;
+  const addressPostcode = informationAttributes
```
Can you refactor the same way you did here for passport controller too
